eth/downloader/queue.go

```diff
@@ -162,18 +168,25 @@ func (q *queue) Reset() {
 	q.resultOffset = 0
 }
 
+func (q *queue) Close() {
```

Comment:
Method doc is missing. Also please mention that it's fine (if it is so) to call this multiple times.